fw/access_log.c

```diff
@@ -266,6 +277,123 @@ process_truncated(TfwStr *in, BasicStr *out, char *p, char *end,
 	}
 }
 
+static void
+do_access_log_req_mmap(TfwHttpReq *req, u16 resp_status,
+		       u32 resp_content_length)
+{
+	u64 *dropped = this_cpu_ptr(&mmap_log_dropped);
+	TfwBinLogEvent *event;
+	unsigned int room_size;
+	TfwStr referer, ua;
+	u32 resp_time;
+	char *data, *p;
+
+	if (!mmap_buffer)
```

Comment:
How mmap_buffer can be equal to zero here? It seems that this function is called only when `access_log_type & ACCESS_LOG_MMAP` and we create mmap_buffer in this case

Reply:
Fixed.